pressure ulcer

```diff
@@ -0,0 +1,77 @@
+-- pressure ulcer during FIRST icu stay 
+ 
+DROP MATERIALIZED VIEW IF EXISTS pressure_ulcer_first_icu ;
+CREATE MATERIALIZED VIEW  pressure_ulcer_first_icu AS
+
+with iepre as 
+(
+select iea.subject_id , iea.hadm_id , iea.icustay_id
+ ,iea.intime
+ ,iea.outtime
+  ,  ROW_NUMBER() over (PARTITION BY iea.subject_id ORDER BY iea.intime) as rn
+from mimiciii.icustays iea 
+)
+
+,ie as 
+(
+select  ieb.hadm_id 
```

Comment:
why do you use `hadm_id` instead of `icustay_id`? obviously you need to retain `hadm_id` in order to join to the notes, but you can retain both columns and drop `hadm_id` at the end of the query.

Reply:
I add the 'hadm_id' and 'icustay_id' , and drop 'hadm_id' at the end of the query